test/parallel/test-string-decoder.js

```diff
@@ -90,7 +90,8 @@ test('utf16le', Buffer.from('3DD84DDC', 'hex'), '\ud83d\udc4d'); // thumbs up
 decoder = new StringDecoder('utf8');
 assert.strictEqual(decoder.write(Buffer.from('E1', 'hex')), '');
 
-// A quick test for lastNeed & lastTotal which are undocumented.
+// A quick test for lastChar & lastNeed & lastTotal which are undocumented.
```

Comment:
nit: `lastChar & lastNeed & lastTotal` -> `lastChar, lastNeed & lastTotal`

Reply:
Thanks. Fixed it.